index.md

```diff
@@ -0,0 +1,8 @@
+--- copyright: years: 2021 - 2023 lastupdated: "2023-04-18" layout: page title: "Home" description: "Welcome to the Open Horizon Documentation web site" nav\_order: 1 ---   
+
+Open Horizon is a platform for managing the service software lifecycle of containerized workloads and related machine learning assets.  It enables autonomous management of applications deployed to distributed webscale fleets of edge computing nodes and devices without requiring on-premise administrators.
+
+Edge computing brings computation and data storage closer to where data is created by people, places, and things.  Open Horizon simplifies the job of getting the right applications and machine learning onto the right compute devices and keeps those applications running and updated.  It also enables the autonomous management of more than 10,000 edge devices simultaneously – that’s 20 times as many endpoints as in traditional solutions.
+
+*   To install everything with a one-liner script, visit the \[Quick Start\](/quick-start/) page.
```

Comment:
Please remove the escaping of the markdown tags.  Instead of "\[Quick Start\](/quick-start/)" it should be `[Quick Start](/quick-start/)`